src/gamepad.h

```diff
@@ -44,10 +44,17 @@ typedef enum GamepadEmulatedButton {
 typedef enum GamepadButton {
 	// must match SDL_GameControllerButton
 	GAMEPAD_BUTTON_INVALID = -1,
+#ifdef __SWITCH__
+	GAMEPAD_BUTTON_B,
+	GAMEPAD_BUTTON_A,
+	GAMEPAD_BUTTON_Y,
+	GAMEPAD_BUTTON_X,
+#else
```

Comment:
I'm not sure, but it looks like this would change the relative physical layout of the default controls (as compared to an xbox-like gamepad), wouldn't it? If so, maybe it'd be better to keep the internal IDs as is, and swap the user-visible names instead?

But I suppose if the default controls are still reasonably comfortable, it doesn't matter that much.

Reply:
They seemed fine but I also did not play more than 2 stages in a row. I'm okay with changing the names instead, it would help the theoretical use case where someone imports its config from PC -> would keep the layout

Reply:
Actually I forgot, changing names would invert Back/Accept in menu actions, so that's why I changed only the enum order, which seemed cleaner. I'd prefer letting this than having enums different in the code than what's actually displayed on the menu, and on the controller, and having to add more ifdef to menu code. Tell me what you think of it

Reply:
Yeah, in that case I think this is fine.